src/_pytest/pytester.py

```diff
@@ -1535,7 +1541,9 @@ def _match_lines(
             pattern
         :param str match_nickname: the nickname for the match function that
             will be logged to stdout when a match occurs
-        :param consecutive: match lines consecutively?
+        :param mode:
+            "consecutive": match lines consecutivly
+            "complete": require all lines to be matched in total
         """
         if not isinstance(lines2, collections.abc.Sequence):
```

Comment:
If we are not using `Enum`, we should then check that `mode in ("consecutive", "complete")`, with a ValueError being raised otherwise?